db/re/mob_db.txt

```diff
@@ -2248,22 +2248,22 @@
 //3088,MM_BRINARANEA_BABY,Mm Brinaranea Baby,Mm Brinaranea Baby,155,155600,1,0,1,1,1,1,80,200,16,26,30,115,79,5,10,12,2,0,20,0x0,300,1000,1000,1000,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 //3089,MM_LOCO_KASA,Mm Loco Kasa,Mm Loco Kasa,155,185000,1,0,0,1,3266,666,30,30,88,44,88,21,95,44,10,12,2,6,47,0x2000000,200,1500,600,500,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 //3090,MM_LOCO_SALAMANDER,Mm Loco Salamander,Mm Loco Salamander,155,217650,1,0,0,1,2678,1257,121,36,60,103,45,35,172,15,10,12,1,6,47,0x2000000,200,1000,500,600,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-3091,MM_BRINARANEA,Brinaranea,Brinaranea,165,81650000,1,0,0,3,8255,197,264,177,135,135,12,72,220,91,10,12,2,6,61,0x6280000,200,1020,500,768,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-3092,MM_MUSPELLSKOLL,Muspellskoll,Muspellskoll,165,55620000,1,0,0,3,9672,98,211,140,202,119,6,45,275,71,10,12,2,6,63,0x6280000,200,608,408,336,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+3091,MM_BRINARANEA,Brinaranea,Brinaranea,165,81650000,1,0,0,3,8255,197,264,177,135,135,12,72,220,91,10,12,2,6,61,0x6280000,200,1020,500,768,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,27324,1
+3092,MM_MUSPELLSKOLL,Muspellskoll,Muspellskoll,165,55620000,1,0,0,3,9672,98,211,140,202,119,6,45,275,71,10,12,2,6,63,0x6280000,200,608,408,336,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,27325,1
 //3093,MM_BRINARANEA_CORE,Mm Brinaranea Core,Mm Brinaranea Core,160,300,1,0,0,0,1,1,1,1,1,1,1,1,1,1,10,12,0,6,21,0x2170000,2000,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 //3094,MM_MUSPELLSKOLL_CORE,Mm Muspellskoll Core,Mm Muspellskoll Core,160,300,1,0,0,0,1,1,1,1,1,1,1,1,1,1,10,12,0,6,23,0x2170000,2000,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 //3095,MM_GOD_SHADOW,Mm God Shadow,Mm God Shadow,1,100000000,1,0,0,0,2,1,1,1,1,1,1,1,1,1,10,12,0,6,23,0x2170000,2000,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-//3096,MM_MOROCC_KID,Mm Morocc Kid,Mm Morocc Kid,175,80000000,1,0,0,3,4980,209,158,134,90,122,7,87,287,36,10,12,0,6,66,0x6280000,200,1000,460,350,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-3097,MM_MOROCC_ADT,Despair God Morocc,Despair God Morocc,175,120000000,1,0,0,3,5523,175,203,155,122,103,12,106,269,51,10,12,1,6,47,0x6280000,200,750,510,500,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+//3096,MM_MOROCC_KID,Mm Morocc Kid,Mm Morocc Kid,175,80000000,1,0,0,3,4980,209,158,134,90,122,7,87,287,36,10,12,0,6,66,0x6280000,200,1000,460,350,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4559,1
+3097,MM_MOROCC_ADT,Despair God Morocc,Despair God Morocc,175,120000000,1,0,0,3,5523,175,203,155,122,103,12,106,269,51,10,12,1,6,47,0x6280000,200,750,510,500,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,27321,1
```

Comment:
If you want to add this change please uncomment 27321 in item DB

Reply:
Done enable that